extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/Metrics.java

```diff
@@ -63,6 +64,7 @@ public DimensionsAndCollector getByName(String name, String service)
 
   public Metrics(String namespace, String path, boolean isAddHostAsLabel, boolean isAddServiceAsLabel)
   {
+    Map<String, DimensionsAndCollector> registeredMetricsTmp = new HashMap<>();
```

Comment:
Nit: It is fine to name this as `registeredMetrics` as you would anyway be referring to the field by using `this.registeredMetrics`.

Reply:
It's OK. There are indeed many places in Java that like this usage style.